packages/wallet-ui/src/services/useStarkNetSnap.ts

```diff
@@ -694,7 +694,7 @@ export const useStarkNetSnap = () => {
           },
         },
       });
-      return response;
+      return response.totalBalance;
```

Comment:
i believe you are trying to do a backward compatible case , so it should be using the latest balance?